src/ScrumBoardItBundle/Controller/DefaultController.php

```diff
@@ -91,13 +97,15 @@ public function printAction(Request $request)
     /**
      * @Route("/flag/add", name="add_flag")
      *
+     * @Secure("has_role('ROLE_AUTHENTICATED')")
+     *
      * @param Request $request            
      * @return Response
      */
     public function addFlagAction(Request $request)
     {
-        $manager = $this->container->get('service.manager');
-        $service = $manager->getService();
+        $service = $this->container->get($this->getUser()
```

Comment:
please use controller shortcuts like $this->get() instead of accessing the container.